standard/expressions.md

```diff
@@ -4033,7 +4034,11 @@ where `x` is an expression of a nullable value type, if operator overload resolu
 
 ### 11.11.11 The is operator
 
-The `is` operator is used to check if the run-time type of an object is compatible with a given type. The check is performed at runtime. The result of the operation `E is T`, where `E` is an expression and `T` is a type other than `dynamic`, is a Boolean value indicating whether `E` is non-null and can successfully be converted to type `T` by a reference conversion, a boxing conversion, an unboxing conversion, a wrapping conversion, or an unwrapping conversion.
+There are two forms of the `is` operator.  One is the *is-type operator*, which has a type on the right-hand-side.  The other is the *is-pattern operator*, which has a pattern on the right-hand-side.
+
+#### 11.11.11.1 The is-type operator
+
+The *is-type operator* is used to check if the run-time type of an object is compatible with a given type. The check is performed at runtime. The result of the operation `E is T`, where `E` is an expression and `T` is a type other than `dynamic`, is a Boolean value indicating whether `E` is non-null and can successfully be converted to type `T` by a reference conversion, a boxing conversion, an unboxing conversion, a wrapping conversion, or an unwrapping conversion.
 
 The operation is evaluated as follows:
```

Comment:
E is no longer permitted to be a method group.  I don't know when that change happened.

Reply:
Two changes should be made sooner rather than later:
1. We should document the error `error CS0837: The first operand of an 'is' or 'as' operator may not be a lambda expression, anonymous method, or method group.` and update this section accordingly.
2. Change the words "of if" below to "or if".

Reply:
Added to #934